<?xml version="1.0"?>
<launch>

  <!-- send parameters to parameter server -->
  <rosparam command="load" ns="map_accessibility_analysis" file="$(find ipa_room_exploration)/ros/launch/cob_map_accessibility_analysis_params.yaml"/>

  <!--  -->
  <node pkg="cob_map_accessibility_analysis" ns="map_accessibility_analysis" type="map_accessibility_analysis_server" name="map_accessibility_analysis" output="log">	<!-- ns=namespace (arbitrary), type=name of executable, name=node name (arbitrary) -->
	<!--launch-prefix="/usr/bin/gdb"-->
	<remap from="map" to="/map"/>
	<remap from="obstacles" to="/move_base/global_costmap/obstacles"/>
	<remap from="inflated_obstacles" to="/move_base/local_costmap/inflated_obstacles"/>
  </node>


</launch>
